code/src/pages/not-found/NotFoundPage.scss

```diff
@@ -0,0 +1,43 @@
+@import "@design-system";
+
+.not-found-page {
+  display: flex;
+  justify-content: center;
+  text-align: center;
+  padding: spacing(8);
+  background-color: var(--spa-not-found-bgc, $light);

```

Comment:
Why `bgc`? We always used `background-color` to be more descriptive

Reply:
> Why `bgc`? We always used `background-color` to be more descriptive

Fixed! Thanks!